firmware/projects/Demo/AnalogInput/main.cc

```diff
@@ -0,0 +1,27 @@
+#include "bindings.hpp"
+
+int main() {
+    bindings::Init();
+
+    while (true) {
+        float volts = bindings::analog_input.ReadVoltage();
+        if (volts <= 1) {
+            bindings::red_Led.SetHigh();
+            bindings::green_Led.SetLow();
+            bindings::blue_Led.SetLow();
+        } else if (volts <= 2) {
+            bindings::red_Led.SetLow();
+            bindings::green_Led.SetLow();
+            bindings::blue_Led.SetHigh();
+        } else if (volts <= 3) {
+            bindings::red_Led.SetLow();
+            bindings::green_Led.SetHigh();
+            bindings::blue_Led.SetLow();
+        } else {
+            bindings::red_Led.SetHigh();
+            bindings::green_Led.SetHigh();
+            bindings::blue_Led.SetHigh();
+        }
```

Comment:
```suggestion
        bool red = false, green = false, blue = false;
        if (volts <= 1) {
            red = true;
        } else if (volts <= 2) {
            blue = true;
        } else if (volts <= 3) {
            green = true;
        } else {
            red = true;
            blue = true;
            green = true;
        }
        bindings::red_Led.Set(red);
        bindings::blue_Led.Set(blue);
        bindings::green_Led.Set(green);
```
This makes it a bit easier to see which volts cause which lights to go on